_IBC/quicksilver-secret.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "quicksilver",
+    "client_id": "07-tendermint-28",
+    "connection_id": "connection-14"
+  },
+  "chain_2": {
+    "chain_name": "secret",
```

Comment:
oh, sorry, I missed this. 'secret' should be 'secretnetwork' as that's the exact chain_name

Reply:
no worries, should have looked at other files my bad- fixed it